src/wiki/plugins/inputs/static/wiki/plugins/inputs/inputs.js

```diff
@@ -0,0 +1,131 @@
+$.ajaxSetup({
```

Comment:
This is essentially a lot of JS code to make it work.

1. It's hard to know what's going on with no comments
1. It won't work without javascript!
1. There is some kind of template rendering going on, it seems redundant when template rendering is something expected to happen through Django?